### ISSUE TITLE
REPL default colours are not accessible

### ISSUE THREAD

**Issue by zooba** [Microsoft/PTVS#2586]
We need to ensure minimum levels of contrast in all our themes

We also need to rename the colour definitions to include "Python", since they are our colours and are not shared with other extensions.
